.circleci/config.templ.yml

```
@@ -573,6 +573,14 @@ jobs:
       - run_test:
           pattern: "datastreams"
 
+  tracer:
+    <<: *machine_executor
+    parallelism: 10
+    steps:
+      - run_test:
+          pattern: "tracer"
+          snapshot: true
```

We need to support the `snapshot=True` configuration in `.gitlab/tests.yml` and then we can avoid adding the tracer test suite back to circleci.

```suggestion
  tracer:
```